.gitignore

```diff
@@ -11,3 +11,4 @@ src/test/resources/cloudant-2.properties
 *.iml
 build/
 .gradle/
+gradle.properties
```

Comment:
I fixed the ignore to be for local.properties in 5803642, but local.properties doesn't get picked up by gradle so isn't the right place to set test configuration. I have updated the instructions in 0679f26 to suggest using GRADLE_USER_HOME for the gradle.properties if you don't want to accidentally commit them.